pylops/signalprocessing/patch2d.py

```diff
@@ -264,7 +264,7 @@ def Patch2D(
             for win_in, win_end in zip(dwin0_ins, dwin0_ends)
         ]
     )
-    Pop = aslinearoperator(combining0 * combining1 * OOp)
+    Pop = LinearOperator(combining0 * combining1 * OOp)
```

Comment:
I believe `LinearOperator` can be removed here. Worked on my local tests.

Reply:
Mmh did you try running the doc?

I think it works up to the point you want to pass x as a Nd-array (non-flattened). At that point it stops working as one operator (I checked and I think it’s either Block or a stack or likely both) in the chain tries to flatten. By wrapping around LinearOperator we basically enforce that at the end of the chain the array is brought back to its correct Nd-array state

Reply:
Got it!